jetpack-io/devbox
# [extension] Generate the same devcontainer files as CLI

## Summary
CLI command `devbox generate devcontainer` generates a Dockerfile that's Alpine based. The vscode extension generates a microsoft-based image and requires finding out if the cpu architecture is arm64 based. 
This change aligns both generated files to be identical and removes the need for differentiating arm64 based image.

## How was it tested?
- Run extension via vscode (run extension action in run & debug)
- Choose "Devbox: Generate Dev Containers config files" from command palette (cmd + shift + p)
- Confirm the .devcontainer files are generated.
- Choose "Dev Containers: Rebuild and Reopen in Container" from command palette.
- Confirm the IDE opens successfully in a container.
